web/profiles/custom/os2loop/themes/os2loop_theme/assets/documents.scss

```diff
@@ -39,6 +39,20 @@ table.loop-documents-table {
   }
 }
 
+th,
```

Comment:
Do we want this on all tables on site or only those within a document?

Reply:
Fixed. Nice catch!